.buildkite/pipeline.yml

```diff
@@ -64,6 +64,7 @@ steps:
   # Run Unit Tests
   #################
   - label: "🔬 :wordpress: Unit Tests"
+    key: "unit_tests_wordpress"
```

Comment:
I'm guessing this was added for consistency? I don't see any previous or new usage

Reply:
Hmm I don't recall adding it, tbh. Maybe this was a leftover from a rebase conflict or something like that?
Anyway, nice catch, will remove it.